<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgPreferences</class>
 <widget class="QDialog" name="dlgPreferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <property name="windowIcon">
   <iconset resource="vtitestmanager.qrc">
    <normaloff>:/images/preferences.png</normaloff>:/images/preferences.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabPreferences">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabGeneral">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QGroupBox" name="gbRunningEnvironment">
         <property name="title">
          <string> Running environment </string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QRadioButton" name="rbEnvCell">
            <property name="text">
             <string>Cell</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rbEnvMinisystem">
            <property name="text">
             <string>Minisystem</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rbEnvStandalone">
            <property name="text">
             <string>Standalone</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rbEnvDesktop">
            <property name="text">
             <string>Desktop</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>251</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gbReleases">
         <property name="title">
          <string> Releases </string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="lblDirLocalReleases">
              <property name="text">
               <string>Local directory :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="ledDirLocalReleases">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="readOnly">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pbDirLocalReleasesSelect">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>25</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>25</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Change directory ...</string>
              </property>
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gbProcesses">
         <property name="title">
          <string> Commands to process </string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <widget class="QLabel" name="lblCommand">
              <property name="text">
               <string>Command :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="ledCommand"/>
            </item>
            <item>
             <widget class="QPushButton" name="pbSave">
              <property name="text">
               <string>Add</string>
              </property>
              <property name="icon">
               <iconset resource="vtitestmanager.qrc">
                <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="lblListCommands">
            <property name="text">
             <string>List of saved commands :</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <item>
             <widget class="QListWidget" name="listCommands"/>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="QPushButton" name="pbAdd">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>Add</string>
                </property>
                <property name="icon">
                 <iconset resource="vtitestmanager.qrc">
                  <normaloff>:/images/import.png</normaloff>:/images/import.png</iconset>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pbModify">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Modify</string>
                </property>
                <property name="icon">
                 <iconset resource="vtitestmanager.qrc">
                  <normaloff>:/images/process.png</normaloff>:/images/process.png</iconset>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pbDelete">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Delete</string>
                </property>
                <property name="icon">
                 <iconset resource="vtitestmanager.qrc">
                  <normaloff>:/images/delete.png</normaloff>:/images/delete.png</iconset>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_4">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabAif">
      <attribute name="title">
       <string>AIF</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="lblDirAIF">
           <property name="text">
            <string>Testinfra directory :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ledDirAIF">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="readOnly">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbDirAIFSelect">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>25</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>25</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Change directory ...</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="lblDirAIFTestConfig">
           <property name="text">
            <string>AIF ATP config file directory :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ledDirAIFTestConfig">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="readOnly">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbDirAIFTestConfigSelect">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>25</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>25</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Change directory ...</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="lblAIFTestcaseTypes">
           <property name="text">
            <string>Testcase types :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ledAIFTestcaseTypes"/>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>299</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabCycler">
      <attribute name="title">
       <string>Cycler</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="lblDirCycler">
           <property name="text">
            <string>Cycler directory :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ledDirCycler">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="readOnly">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbDirCyclerSelect">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>25</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>25</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Change directory ...</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="lblDirCyclerTestConfig">
           <property name="text">
            <string>Cycler ATP config file directory :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ledDirCyclerTestConfig">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="readOnly">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbDirCyclerTestConfigSelect">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>25</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>25</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Change directory ...</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbDirCyclerTestConfigDerive">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>25</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>25</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Derive directory from 'Cycler directory'</string>
           </property>
           <property name="text">
            <string>&lt;&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="lblCyclerTestcaseTypes">
           <property name="text">
            <string>Testcase types :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ledCyclerTestcaseTypes"/>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>299</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbApply">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Apply</string>
       </property>
       <property name="icon">
        <iconset resource="vtitestmanager.qrc">
         <normaloff>:/images/ok.png</normaloff>:/images/ok.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbClose">
       <property name="text">
        <string>Close</string>
       </property>
       <property name="icon">
        <iconset resource="vtitestmanager.qrc">
         <normaloff>:/images/close.png</normaloff>:/images/close.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="vtitestmanager.qrc"/>
 </resources>
 <connections/>
</ui>
